deployments/deployment/appprotect-dos-arb.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: appprotect-dos-arb
+  namespace: nginx-ingress
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: appprotect-dos-arb
+  template:
+    metadata:
+      labels:
+        app: appprotect-dos-arb
+    spec:
+      containers:
+        - name: appprotect-dos-arb
+          image: docker-registry.nginx.com/nap-dos/app_protect_dos_arb:latest
```

Comment:
do we need to reference a version here rather than the latest?

The IC support policy is described here -- https://docs.nginx.com/nginx-ingress-controller/technical-specifications/#supported-nginx-ingress-controller-versions . We support a release for 2 years.

Let's imagine a situation that somebody deploys this file as part of 1.13 release half year of a year after the release 1.13. Will the DOS software for NGINX Plus R25  will still work with the latest arbitrator?

Reply:
"Not change a lot at all" - we cannot tell the future, and specifying a version protects us against that. If best design is to specify a version then we should reference a version. Good design protects us against our own incorrect assumptions.